tests/shipping/models/Delivery - Second Purchase - Without Geolocation and PostalCode - Boleto.model.js

```diff
@@ -0,0 +1,35 @@
+import { setup, visitAndClearCookies } from "../../../utils"
+import {
+  fillEmail,
+  confirmSecondPurchase,
+  getSecondPurchaseGeolocationEmail,
+} from "../../../utils/profile-actions"
+import { completePurchase, payWithBoleto } from "../../../utils/payment-actions"
+
+export default function test(account) {
+  describe(`Delivery - 2P - Boleto - ${account}`, () => {
```

Comment:
It might be worth mentioning "without geolocation and postal code" here, even though tests are usually identified by file name.

Reply:
good catch!